clientSide/womeneducation/src/App.js

```diff
@@ -1,12 +1,44 @@
-import logo from './logo.svg';
-import './App.css';
+// import HomepageBackend from'./Components/Home/HomeBackEnd';
+//  import React from 'react';
+//  import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+// import Welcomepage from './Components/Home/Welcomepage';
+
+
+// function App() {
+//   return (
+//     <div className="App">
+     
+//       {/* <HomepageBackend/> */}
+//       {/* <Welcomepage/> */}
+//        <Router>
+//         <Switch>
+//           <Route path="/homepage" component={HomepageBackend} />
+//           <Route path='/welcomepage' component={Welcomepage} />
+//         </Switch>
+//       </Router> 
+//     </div> 
+//   );
+// }
+
+// export default App;
+
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import HomepageBackend from './Components/Home/HomeBackEnd';
+import Welcomepage from './Components/Home/Welcomepage';
 
 function App() {
   return (
     <div className="App">
-      pavani
+      {/* Other components */}
+      <Router>
+        <Routes>
+          <Route path="/homepage" element={<HomepageBackend />} />
```

Comment:
Routes should be in Route.jsx not in app.js

Reply:
I created Routes folder separately.